flink-table/flink-table-common/src/main/java/org/apache/flink/table/utils/DateTimeUtils.java

```diff
@@ -386,6 +406,18 @@ public static TimestampData toTimestampData(DecimalData v, int precision) {
         }
     }
 
+    public static TimestampData toTimestampData(long epoch) {
+        return toTimestampData(epoch, 3);
+    }
+
+    public static TimestampData toTimestampData(double epoch) {
+        return toTimestampData(epoch, 3);
+    }
+
+    public static TimestampData toTimestampData(DecimalData epoch) {
+        return toTimestampData(epoch, 3);
+    }
+
```

Comment:
from any user outside of `TO_TIMESTAMP_LTZ` 3 here is a magic number.
I would suggest to have such calls only in your toTimestampLtzFunction class and don't put them here